packages/path_provider/path_provider_linux/example/linux/Makefile

```diff
@@ -0,0 +1,146 @@
+include app_configuration.mk
```

Comment:
The `example/linux/` folder must not be checked in; it will break. The test driver will create it on the fly.